pkg/daemon/daemon.go

```diff
@@ -428,7 +428,10 @@ func (d *Daemon) syncConsul() error {
 						meta[k] = v
 					}
 				}
-
+				serviceStatus := status
+				if meta[MetaAlwaysHealthy] == "true" {
```

Comment:
IIRC the other uses do a `ParseBool` in case people have things like `True` `False` `false` etc.

Reply:
done